src/lib/services/tx/index.ts

```diff
@@ -53,15 +53,17 @@ export const useTxData = (
 
       const logs = extractTxLogs(txResponse);
 
+      const payload = snakeToCamel(txResponse);
+
       return {
-        ...txResponse,
+        ...payload,
         timestamp: parseDateOpt(txResponse.timestamp),
         logs,
         chainId: currentChainId,
         isTxFailed: Boolean(txResponse.code),
       };
     },
-    [endpoint, currentChainId, isFullTier]
+    [currentChainId, endpoint, isFullTier]
```

Comment:
Only endpoint should be enough.

Reply:
If we use only `endpoint` parameter, the Eslint will report the error.